python/tests/integration/completions/test_chat_completions.py

```diff
@@ -47,6 +49,13 @@
 except KeyError:
     mistral_ai_setup = False
 
+ollama_setup: bool = False
+try:
+    if os.environ["OLLAMA_MODEL"]:
```

Comment:
Can't you just use os.getenv?

Reply:
I am following what's been done for Mistral. What is the benefit of using os.getenv over os.environ? We are just trying to see if the environment variable is set, not trying to retrieve the value.

Reply:
os.getenv doesn't throw, but you can set a default (like with a dict `.get('key', 'default')` vs `['key']`)

Reply:
and change the mistral one while you're here :)